cluster/src/shard_lock_manager.rs

```diff
@@ -400,28 +518,30 @@ impl ShardLock {
             return Ok(false);
         }
 
-        // Grant the lease first.
-        let resp = etcd_client
-            .lease_grant(self.ttl_sec as i64, None)
-            .await
-            .context(GrantLease {
-                shard_id: self.shard_id,
-            })?;
-        ensure!(
-            resp.ttl() > 0,
-            GrantLeaseWithInvalidTTL {
-                shard_id: self.shard_id,
-                ttl_sec: resp.ttl()
+        let lease_info = match self.maybe_fast_acquire_lock(etcd_client).await {
+            Ok(Some(v)) => {
+                info!("Shard lock is acquired fast, shard_id:{}", self.shard_id);
```

Comment:
I think maybe a concrete prefix for log in a specific module is useful?
When debugging, I found we always need to grep a series of logs in a module rather than just a single log, and that can be convenient by adding prefix?

Reply:
I guess the file name in the log can meet your requirements, and manual prefix is verbose.